src/Bridge/Symfony/Bundle/Test/Client.php

```diff
@@ -212,4 +212,25 @@ public function enableReboot(): void
     {
         $this->kernelBrowser->enableReboot();
     }
+
+    /**
+     * Extract headers depending on the symfony/http-client version being used.
+     */
+    private static function extractHeaders(array $options): array
+    {
+        if (!isset($options['normalized_headers'])) {
+            return $options['headers'];
+        }
+
+        $headers = [];
```

Comment:
```suggestion
        /** @var array<string, string[]> $headers */
        $headers = [];
```

Looks like this should make phpstan happy. See https://github.com/phpstan/phpstan/issues/2404